public/pages/Snapshots/components/RenameInput/RenameInput.tsx

```diff
@@ -24,47 +28,53 @@ const RenameInput = ({ getRenamePattern, getRenameReplacement }: RenameInputProp
     setRenameReplacement(e.target.value);
     getRenameReplacement(e.target.value);
   };
+  const patternText = `Define the format of renamed indices. Use $0 to include the \n
+  entire matching index name, $1 to include the content of the first\n
+  capture group, etc.`;
 
-  return (
-    <>
-      <EuiSpacer size="l" />
+  const renameText = `Define the format of renamed indices. Use $0 to include the\n
+  entire matching index name, $1 to include the content of the first\n
+  capture group, etc.`;
 
+  const patternLabel = (
+    <>
       <EuiText size="xs">
         <h4>Rename Pattern</h4>
       </EuiText>
-      <EuiText size="xs" style={{ padding: "0px 0px 5px 0px" }}>
-        <p style={{ fontWeight: 200 }}>
-          Use regular expression to define how index names will be renamed.
-          <br />
-          By default, input (.+) to reuse the entire index name.{" "}
-          <EuiLink href={RESTORE_SNAPSHOT_DOCUMENTATION_URL} target="_blank" rel="noopener noreferrer" external={true}>
-            Learn more
-          </EuiLink>
-        </p>
+      <EuiText size="xs" style={{ padding: "0px 0px 5px 0px", fontWeight: "200" }}>
+        {patternText}{" "}
```

Comment:
Can we use a single template string here? `${patternText} `

Reply:
sure.  updated for next PR